test/bin/import.spec.js

```diff
@@ -15,7 +15,6 @@ describe('import CLI tests', function () {
         importPath = './test/bin/import/project.webgmex',
         projectName,
         projectId,
-        __should = testFixture.should,
```

Comment:
If you run this test file on its own, line 100 will fail. Since that's the only should, you can just change it to expect..